custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -588,6 +654,10 @@ function DownloadController($scope, $rootScope, $location, locationHierarchy, lo
         return !(vm.isDashboardUsageSelected() || vm.isAwwActivityReportSelected());
     };
 
+    vm.showQuarterFilter = function() {
+        return !vm.isDashboardUsageSelected() && vm.isQuarterDataPeriodSelected();
```

Comment:
why does it depend on the dashboard usage report?

Reply:
because that report doesn't required any kind of month related filter

Reply:
I may not be following this but do other reports need a quarter filter?

Reply:
that s handled by the dataPeriod

Reply:
hey, I am actually not following, why `!vm.isDashboardUsageSelected()` this is required here? 
is not just  `isQuarterDataPeriodSelected`  enough?